cloudsql/mysql/database-sql/README.md

```diff
@@ -15,9 +15,9 @@ Note the connection string, database user, and database password that you create
 [instructions](https://cloud.google.com/sql/docs/mysql/create-manage-databases).
 Note the database name.
 
-1. Create a service account with the 'Cloud SQL Client' permissions by following these
-[instructions](https://cloud.google.com/sql/docs/mysql/connect-external-app#4_if_required_by_your_authentication_method_create_a_service_account).
-Download a JSON key to use to authenticate your connection.
+1. Set up [Application Default Credentials][adc]
```

Comment:
@enocom probably still worth highlighting the need of the `Cloud SQL Client` IAM role?

Reply:
Done.